jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -342,7 +292,7 @@ def _uncertainty(result):
                 # TODO: update specutils to be consistent with region vs regions and default to
                 # regions=None so this elif can be removed
                 temp_result = FUNCTIONS[function](spec_subtracted, region=None)
-                self.results_centroid = temp_result.to_value(u.AA)
+                self.results_centroid = temp_result.to_value(u.AA, equivalencies=u.spectral())
```

Comment:
ran into a bug when testing when the spectrum was in frequencies

Reply:
Is this bug out of scope here? If so, is there a follow-up issue?